### DISCUSSION TITLE
About Twill on Spectrum

### DISCUSSION THREAD

**Discussion by ifox** [area17/twill#888] [General]
Hey there,

Many of you have asked for a space to chat with us and other developers using Twill. We just launched [Twill on Spectrum](https://spectrum.chat/twill) to do just that. We literally just created it so it’s a bit of a graveyard right now – don’t be shy and post away to help get things started!

We also released a lot more today, so I invite you to check it out there! With the 1.2.1 release out we're going to have more time to reply to bug reports and feature requests here but would appreciate if we all use Spectrum more for things that are more conversations/questions than actual issues!

Hope you like it!

Quentin

**Comment by kirkbushell** [comment]
It's still a graveyard.

**Comment by IllyaMoskvin** [comment]
Looks like Spectrum will become read-only August 10, 2021.

https://spectrum.chat/spectrum/general/e4ca0386-f15c-4ba8-8184-21cf5fa39cf5

**Comment by lostdesign** [comment]
Whats stopping Area17 from opening the discussions tab here on github right away? While this feature is labeled as beta, any repo can activate it and its pretty much stable to use.

The only changes that are coming, are additions such as GraphQL API to do CRUD on discussions. May I further more suggest, opening a Discord channel? If you'd like, I could support you on that and take the lead.

I am running several discord communities, one of which is for webdevelopers that accounts 23k members as of now.

The combination of github discussions for thread like things and discord for real time discussions, will surely improve the community overall. 

I'd simply not wait out on for so long, if time is the issue, I'd be here to help drive the community towards the right direction.

@ifox @m4n1ok @antonioribeiro

**Comment by ifox** [comment]
Hi @lostdesign, we're on the same page. With the recent announcement from Spectrum, we're going to move forward with GitHub Discussions, and we're opening a Discord too. It's already created and we're setting it up at the moment before announcing it to the community with our upcoming release. Glad to know we have your support!

**Comment by lostdesign** [comment]
@ifox Feel free to invite or contact me anytime, I am `lost#0001` on discord.

**Comment by kirkbushell** [comment]
> Hi @lostdesign, we're on the same page. With the recent announcement from Spectrum, we're going to move forward with GitHub Discussions, and we're opening a Discord too. It's already created and we're setting it up at the moment before announcing it to the community with our upcoming release. Glad to know we have your support!

Am onboard also - hopefully we can get the community more active.

When are you guys looking at the next release? Really interested to see where this project goes - also happy to sponsor/pay for the product, I think you guys have a great foundation here but it still needs a lot of work to get it to a point where someone can pick it up and fly forward :)

**Comment by lostdesign** [comment]
Any news on the discord?

**Reply by ifox** [reply]
Hi @lostdesign, sorry for the delay, catching up with issues and discussions now. 

We've been focusing on reviewing and merging the work of the community to get back on our feet to accept new contributions, and finally released 2.2.0 last week. 

Check out our full announcement on Twitter: https://twitter.com/twillcms/status/1394337845091962880?s=19.

Discord is open at: https://discord.link/twill.

Cheers.

**Reply by ConstantMath** [reply]
Hi @ifox, why choosing a proprietary app like Discord.
They are alternatives like Riot (Element) that make more sense imo since we all try to make free licence source code ..well I think so.

**Reply by lostdesign** [reply]
I think the chat is the least thing that should matter here in terms of its licensing or wether it is closed or not. Maintaining and hosting a chat is hassle enough, so a service, that is free and works quite well, surely is a good plan to use then.

Futher more, discord is such a huge platform, which basically is free marketing if done right.

**Reply by ConstantMath** [reply]
I rather think that's the first thing to consider, especially for what you just said.

**Reply by pboivin** [reply]
An interesting discussion... Perhaps it's more about convenience and less about personal values. I mean, we all value freedom and openness but, at the end of the day, GitHub (also proprietary & closed source) is still an amazing platform to collaborate on code and documentation. I believe the same applies to Discord.
